backend/src/main/java/ba/unsa/etf/si/bbqms/domain/TenantLogo.java

```diff
@@ -0,0 +1,60 @@
+package ba.unsa.etf.si.bbqms.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+
+@Entity
+public class TenantLogo {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "base64_logo")
+    private String base64Logo;
+
+    @OneToOne(mappedBy = "logo")
+    @JsonIgnore
+    private Tenant tenant;
```

Comment:
Do we really need to have reference to a tenant in the logo? As far as I see we will never be only fetching the logo, but rather fetching it as a part of the tenant data, so there is no need to know to which tenant it belongs to.

Reply:
True, a one-to-one unidirectional relationship is sufficient. Implemented changes.